src/governance-vote.h

```diff
@@ -140,7 +140,7 @@ class CGovernanceVote
     uint256 GetHash() const
     {
         CHashWriter ss(SER_GETHASH, PROTOCOL_VERSION);
-        ss << vinMasternode;
+        ss << masternodeOutpoint << uint8_t{} << 0xffffffff;
```

Comment:
Any chance to get rid of this? Or at least add a comment about why it is there (I understand why it's there but only because of the PRs context).

Reply:
Can't easily get rid of it because this would break the hash and we have to keep votes for a long time. Could have an additional comment mentioning that we write these dummy values for legacy reasons, agree. Will add comments like that in another (cleanup) PR later.